addon/components/ce/content-editable.ts

```diff
@@ -332,7 +300,8 @@ export default class ContentEditable extends Component {
           break;
         }
       }
-      this.rawEditor.generateDiffEvents.perform();
+      // eslint-disable-next-line @typescript-eslint/unbound-method
+      void taskFor(this.rawEditor.generateDiffEvents).perform();
       this.rawEditor.model.read();
```

Comment:
just a question: why do we need to read here?

Reply:
not all handlers work on the vdom, so we have to read to update the model with the changes in the vdom. once all handlers work on the vdom this should no longer be needed.